src/essentials/Colors/Colors.ts

```diff
@@ -5,6 +5,17 @@ import { createThemeGlobalStyle } from './globalStyles';
 export const Colors = {
     white: 'hsl(0, 0%, 100%)',
     black: 'hsl(0, 0%, 0%)',
+    neutral: {
```

Comment:
These are the "classic" (old) bare colors, I'm not sure if these neutrals make sense here, I understood they belong to the new palette only 🤔

Reply:
Hey @martimalek! Good catch!

In one of the previous iterations this was the suggestion from design that we introduce `neutral` colors to the old classic theme. However now that we agreed to not use any of them I removed them again in 6f84898.